Makefile

```diff
@@ -162,6 +168,12 @@ st-ssl: run-etcd-ssl dist/calicoctl docker calico_test/.calico_test.created busy
 	           calico/test \
 	           sh -c 'cp -ra tests/st/* /tests/st && cd / && nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer $(ST_OPTIONS)'
 
+add-ssl-hostname:
```

Comment:
The certificate requires a specific hostname to be used.  The tool for creating certificates uses json files that have the hostname hardcoded.  We could potentially have a template that creates a json file using your hostname, but I think this is fine.